classwork 4.10/classwork 4.10/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace hometask2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double V;
+            Console.WriteLine("Введите значение длины стороны основания правильной треугольной пирамиды");
+            var side = double.Parse(Console.ReadLine());
+            Console.WriteLine("Введите значение высоты правильной треугольной пирамиды");
+            var high = double.Parse(Console.ReadLine());
```

Comment:
high - это высокий, высота пишется иначе
```suggestion
            var height = double.Parse(Console.ReadLine());
```